README.md

```diff
@@ -90,6 +90,33 @@ Downloading CSVs to data...
 Writing metadata to data/metadata.tsv...
 ```
 
+## 🚆 Command line interface
+
+``` bash
+$ airbase download --help
+```
+
+``` man
+Usage: airbase download [OPTIONS]
+
```

Comment:
I think the formatting on GitHub looks nicer with 
```suggestion
``` console
$ airbase download --help
Usage: airbase download [OPTIONS]
```